### ISSUE TITLE
Use data of ditortableImage to load the image in ol map

### ISSUE THREAD

**Issue by mdellanave** [publiclab/Leaflet.DistortableImage#732]
I would like to use the data generated by the DistortableImage (corners, center etc) to create an equivalent map but using ol API instead of Leaflet one (beacause of the possibility of view rotation in ol).
Is it possibile and how?
Can some please give me a hint?

**Scope**:
API, documentation, installation, code, meta, other?

**Description**:
Feel free to include any applicable: code, screenshots, or gifs.

**Comment by mdellanave** [comment]
I'm trying using ol-ext with GeoImage but the problem is that distortableImage uses the image corners instead of  the scale factor used by ol.

Assuming that distortAction will be disabled, is there a way to get the scale factor from the distortableImage api?
O otherwise a way to calculate it?

thank you

**Comment by sashadev-sky** [comment]
@mdellanave I'll open a feature request to add a `getScale` method to the API.

**Comment by mdellanave** [comment]
> @mdellanave I'll open a feature request to add a `getScale` method to the API.

thank you @sashadev-sky!!!

**Comment by rodrigore** [comment]
@mdellanave have you found a solution to use something like this plugin but for open layers?

**Comment by mdellanave** [comment]
> @mdellanave have you found a solution to use something like this plugin but for open layers?

Yes I'm currently using this https://viglino.github.io/ol-ext/examples/interaction/map.interaction.transform.html and this https://viglino.github.io/ol-ext/examples/layer/map.geoimage.html

**Comment by rodrigore** [comment]
> > @mdellanave have you found a solution to use something like this plugin but for open layers?
> 
> Yes I'm currently using this https://viglino.github.io/ol-ext/examples/interaction/map.interaction.transform.html and this https://viglino.github.io/ol-ext/examples/layer/map.geoimage.html

😮 how do you mix the transform with the geoImage? I'm kinda noob of open layers.

I would like to do the distortion of any type of image.

Do you have some example? I would really appreciate it

**Comment by mdellanave** [comment]
[image-map-editor.zip](https://github.com/publiclab/Leaflet.DistortableImage/files/9685802/image-map-editor.zip)

Try with this one! ;-)
It is the best result I have been able to achieve.

cheers

**Comment by rodrigore** [comment]
> [image-map-editor.zip](https://github.com/publiclab/Leaflet.DistortableImage/files/9685802/image-map-editor.zip)
> 
> Try with this one! ;-) It is the best result I have been able to achieve.
> 
> cheers

Wow thank you so much for the example and time. I tested and is working fine. The only 'issue' is that I tried to set keepAspectRatio to false and is not working, but I assume that is harder or the demo doesn't handle that case. Anyway, thank you again 🙂
